docs/sources/query/log_queries/_index.md

```diff
@@ -463,6 +463,27 @@ The **logfmt** parser can operate in two modes:
     "fwd_ip" => "124.133.124.161"
     ```
 
+logfmt parser also supports `--strict` flag to enable strict parsing.
+
+```
+| logfmt --strict
+| logfmt --strict host, fwd_ip="fwd"
+```
+
+With strict parsing enabled, logfmt parser stops scanning the log line and returns early with an error if it encounters any poorly formatted key=value pair.
+```
+// accepted
+key=value key="value in double quotes" standalone_key
+
+// invalid
+=value // no key
+foo=bar=buzz
+fo"o=bar
+```
+
+By default, the parser skips the invalid tokens and continues parsing the rest of the log line.
```

Comment:
"By default" is unclear here, do you mean "when using the default settings" or more that skipping the invalid tokens and continuing to parse is the default behavior?

Reply:
@JStickler I was referring to the latter. I have updated the doc to make this clear.

Reply:
The revision is much clearer.  Thanks!